underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -563,13 +563,8 @@ public InputStream open(String path, OpenOptions options) throws IOException {
     while (retryPolicy.attempt()) {
       try {
         FSDataInputStream inputStream = hdfs.open(new Path(path));
-        try {
-          inputStream.seek(options.getOffset());
-        } catch (IOException e) {
-          inputStream.close();
-          throw e;
-        }
-        return new HdfsUnderFileInputStream(inputStream);
+        // For remote HDFS, use positionedRead instead of seek
```

Comment:
Perhaps change this comment to say why positionedRead is better than seeking in some cases?

Reply:
done